Encounter/encounter_type.md

```diff
@@ -0,0 +1,97 @@
+# Encounter Type
+
+## Overview
+
+* A visit can be associated with one or more encounters. OpenMRS allows you to define multiple encounter types.
```

Comment:
I'd suggest changing this to

> Encounters represent an interaction between the patient and healthcare system. Since there are a wide variety of ways in which these interactions may occur, OpenMRS allows you categorize them by defining different types of encounter. For example, "Adult Primary Care Initial Visit" and "In-Between Visit Documentation" could be different types of encounters.